SECURITY.md

```diff
@@ -1,24 +0,0 @@
-- [Security Policy](#security-policy)
```

Comment:
Please keep the file with as a simple content a redirection towards the security policy of the main repo.